e2e/components/Tabs/Tabs-test.avt.e2e.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { expect, test } = require('@playwright/test');
+const { visitStory } = require('../../test-utils/storybook');
+
+test.describe('Tabs @avt', () => {
+  test('accessibility-checker @avt - Tabs default state', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--default',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs');
+  });
+
+  test('accessibility-checker tabs contained', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--contained',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs-contained');
+  });
+
+  test('accessibility-checker tabs contained full width', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--contained-full-width',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--contained-full-width');
+  });
+
+  test('accessibility-checker tabs contained with icons', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--contained-with-icons',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs-contained-with-icons');
+  });
+
+  test('accessibility-checker tabs contained with secondary labels', async ({
+    page,
+  }) => {
+    await visitStory(page, {
+      component: 'Slider',
+      id: 'components-tabs--contained-with-secondary-labels',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations(
+      'Tabs-contained-with-secondary-labels'
+    );
+  });
+
+  test('accessibility-checker tabs contained with secondary labels and icons', async ({
```

Comment:
We can probably just keep this test and remove the previous two variants that just have secondary labels or just have icons.